raiden/api/python.py

```diff
@@ -414,6 +415,25 @@ def channel_open(
 
         return channel_state.identifier
 
+    def mint_token(
+        self,
+        token_address: typing.TokenAddress,
+        to: typing.Address,
+        value: typing.TokenAmount,
+        method: str,
+    ) -> typing.TransactionHash:
+        """ Try to mint `value` units of the token at `token_address` and assign them to `to`,
+        using the minting method named `method`.
+
+        Raises:
+            TransactionNotSuccessful if the minting fails for any reason.
```

Comment:
I think exceptions should be more specific. `MintFailed` should be introduced/used.